lib/opensusebasetest.pm

```diff
@@ -553,6 +553,12 @@ sub reconnect_s390 {
         type_line_svirt "root", expect => 'Password';
         type_line_svirt "$testapi::password";
         type_line_svirt "systemctl is-active network", expect => 'active';
+        if ((is_alp || is_sle_micro) && is_s390x) {
```

Comment:
```suggestion
        if ((is_alp || is_sle_micro('>=6.0')) && is_s390x) {
```

Reply:
older versions of SLE Micro don't need this.

Reply:
after this is scope we can merge

Reply:
Thanks, fixed